src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ayamiss.cpp

```diff
@@ -180,8 +180,7 @@ struct boss_ayamiss : public BossAI
                 {
                     DoCast(target, SPELL_PARALYZE, true);
                     instance->SetGuidData(DATA_PARALYZED, target->GetGUID());
-                    uint8 index = urand(0, 1);
-                    me->SummonCreature(NPC_HIVEZARA_LARVA, LarvaPos[index], TEMPSUMMON_TIMED_OR_CORPSE_DESPAWN, 30000);
+                    DoCastAOE(RAND(SPELL_SUMMON_LARVA_A, SPELL_SUMMON_LARVA_B), true);
```

Comment:
Something seems to have happened here, merge conflict probably went wrong, not sure if it is just me

![image](https://user-images.githubusercontent.com/47818697/184444744-1274d8f8-a992-4558-ab32-797c89434577.png)